tests/plugins/button/buttonicon.js

```diff
@@ -0,0 +1,225 @@
+/* bender-tags: editor */
```

Comment:
There is no test for adding custom icon for standard buttons (e.g. `Bold`). It should be possible by overwriting `button.icon` property before rendering.